docs/src/transformers.md

```diff
@@ -193,18 +193,17 @@ K-means clustering algorithm assigns one of three labels 1, 2, 3 to
 the input features of the iris data set and compares them with the
 actual species recorded in the target (not seen by the algorithm).
 
-```julia
-import Random.seed!
-seed!(123)
+```julia-repl
+julia> import Random.seed!
+julia> seed!(123)
 
-X, y = @load_iris;
-KMeans = @load KMeans pkg=ParallelKMeans
-kmeans = KMeans()
-mach = machine(kmeans, X) |> fit!
+julia> X, y = @load_iris;
+julia> KMeans = @load KMeans pkg=ParallelKMeans
+julia> kmeans = KMeans()
+julia> mach = machine(kmeans, X) |> fit!
 
-# transforming:
-Xsmall = transform(mach);
-selectrows(Xsmall, 1:4) |> pretty
+julia> # transforming:
```

Comment:
This is really weird: a comment at the `julia>` prompt, here and later. Let's just skip all the prompts that don't precede a block of output (see previous comment).

Reply:
Yes, I'm not particularly happy with this change either, I just did it for consistency. Maybe there's a better way to write this?

Reply:
Maybe a better way to do this would be to split the code sample into 3, one for setup, one for transforming, and one for predicting?

Reply:
Yes, you can re-organize so that the code comments become ordinary markdown text between seperate fenced blocks.